Makefile

```diff
@@ -36,8 +43,9 @@ gofmt:
 	@./hack/verify-gofmt.sh
 
 cri-containerd: check-gopath
-	$(GO) build -o $(BUILD_DIR)/$@ \
-		$(PROJECT)/cmd/cri-containerd
+	$(GO) -o $(BUILD_DIR)/$@ \
+	-ldflags '-w -extldflags "-static" -X $(PKG)/pkg/version.version=$(VERSION)' \
```

Comment:
We don't need other compile flags for now.

I believe `-w` can be kept, but "-static" can be removed.